dts/riscv/it8xxx2-wuc-map.dtsi

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
```

Comment:
I think you need to delete the "All Rights Reserved" to fix the license check.

Reply:
I had tried this before, I rebase the commits and try again, it still  isn't apache-2.0 licensed and has non-permissive license.

Actually other ITE files all have the "All Rights Reserved" string.

I confirm that the copyright format and short identifier is same as Standard License Header, defined in: 
https://spdx.org/licenses/Apache-2.0.html